tools/generators/shared-python-app/generator.ts

```
@@ -0,0 +1,141 @@
+    addModuleSpecifier: options.addModuleSpecifier,
+  });
+
+  const projectName = options.name; // Nx will derive this
+  const projectRoot = options.directory
+    ? joinPathFragments(options.directory, options.name)
+    : `apps/${options.name}`;
+
+  const projectConfigPath = joinPathFragments(projectRoot, "project.json");
+  updateJson(tree, projectConfigPath, (json) => {
+    if (json.targets) {
+      delete json.targets.lint;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)